gecosws_config_assistant/controller/ConnectWithGecosCCController.py

```diff
@@ -672,18 +672,21 @@ def connect(self):
         ou = gecosCC.search_ou_by_text(self.view.get_gecos_access_data(), 
                                   workstationData.get_ou())
         
-        ok, reason = gecosCC.register_computer(self.view.get_gecos_access_data(), 
-                workstationData.get_node_name(), ou[0][0])
+        if not gecosCC.register_computer(self.view.get_gecos_access_data(), 
```

Comment:
Why do not include 'reason' returned value in order to show it to user in case of error?